src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1260,7 +1527,7 @@ protected override BoundExpression VisitExpressionWithoutStackGuard(BoundExpress
         {
```

Comment:
>{ [](start = 8, length = 1)

Please assert `_analyzedNullabilityMapOpt` does not contain an entry for `node` to ensure we are not visiting expressions multiple times.

Reply:
We can't do this: currently, we're explicitly visiting arguments twice.

---
In reply to: [266115954](https://github.com/dotnet/roslyn/pull/34017#discussion_r266115954) [](ancestors = 266115954)